### DISCUSSION TITLE
✨ [Feature]: 使用yorkie替换husky管理工程git提交

### DISCUSSION THREAD

**Discussion by gumingWu** [DevCloudFE/vue-devui#1372] [General]
### What problem does this feature solve

1. 目前工程是使用husky进行git规范控制的，但由于现在是使用了monorepo的形式改造工程，所以我认为husky放在packages/devui-vue是不合理的
2. 尤雨溪开源的工具yorkie，使用脚本的方式来对git钩子进行控制，并且能够移除现有的husky目录，使工程看起来更简洁

![image](https://user-images.githubusercontent.com/24989299/146559439-c3186f94-24e1-4adf-9356-793d8ec0d391.png)

### What does the proposed API look like

1. 移除husky相关内容
2. 使用yorkie进行管理

**Comment by TinsFox** [comment]
feature1 已经处理 #20 
feature2 考虑收纳进入 devui-cli 中🤔
